src/note.rs

```diff
@@ -53,6 +60,25 @@ pub struct Note {
 }
 
 impl Note {
+    /// Generates a dummy spent note.
+    ///
+    /// Defined in [Zcash Protocol Spec § 4.8.3: Dummy Notes (Orchard)][orcharddummynotes].
+    ///
+    /// [orcharddummynotes]: https://zips.z.cash/protocol/nu5.pdf#orcharddummynotes
+    pub(crate) fn dummy(rng: &mut impl RngCore, rho: Option<Nullifier>) -> (FullViewingKey, Self) {
+        let fvk: FullViewingKey = (&SpendingKey::random(rng)).into();
+        let recipient = fvk.default_address();
+
+        let note = Note {
+            recipient,
+            value: NoteValue::default(),
```

Comment:
I really dislike the use of `default` here; can this be `zero()`?

Reply:
Sure, I can add an alias.